### DISCUSSION TITLE
How to export Dutch Tax Report?

### DISCUSSION THREAD

**Discussion by tural-ali** [teslamate-org/teslamate#4138] [General]
Hello everyone, 

I got two questions:
1. How can I export the Dutch tax report into a CSV/XLS file?
2. How do you manually add missing drives to the database?

Thank you.

**Comment by cwanja** [comment]
1. Go to the panel, click on it and go to Inspect > Data > Download CSV
![image](https://github.com/user-attachments/assets/8a4b8ce8-4322-4710-8579-15f9c4cc886f)
2. You would need to insert the row into the drives table.
